build/containers/cbase/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:buster-slim
+
+ENV LC_ALL=C.UTF-8
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        python-scipy python-statsmodels python-mpmath && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
python --> 2.7.16 (This is as of now, although this might need an update)
python-scipy --> 1.1.0
python-statsmodels --> 0.8.0
python-mpmath --> 1.1.0